src/components/routes/Categories.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../../redux/categories/Categories';
 
 const Categories = () => {
-  const handleClick = () => {
-    // console.log('Under construction');
-  };
+  const dispatch = useDispatch();
+  // const handleClick = () => {
+  //   // console.log('Under construction');
+  // };
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧